src/javascripts/ng-admin/Crud/misc/Compile.js

```diff
@@ -28,6 +28,10 @@ define(function () {
 
                         // compile the new DOM and link it to the current scope.
                         $compile(element.contents())(scope);
+
+                        if (attrs.compileOnce == 'true') {
```

Comment:
To be documented (and not really related to this PR - I added that when I experimented another implementation)